introductory/src/App.jsx

```diff
@@ -1,8 +1,14 @@
+import List from "./listComponent"
 
 
 function App() {
+
+  const arrayOfAnimals = ['dog', 'cat', 'chicken', 'cow', 'sheep', 'horse'];
```

Comment:
Be **consistency** in choosing which **type of quotation** mark to use between **" "** and **' '**